code/game/objects/structures/window.dm

```diff
@@ -184,12 +184,19 @@
 		else if(!reinf || (static_frame && state == 0))
 			deconstruct(TRUE)
 
-	else if(iscrowbar(I) && reinf && state <= 1 && deconstructable)
+	else if(iscrowbar(attacking_item) && reinf && state <= 1 && deconstructable)
 		dismantle = TRUE
 		state = 1 - state
 		playsound(loc, 'sound/items/crowbar.ogg', 25, 1)
 		to_chat(user, (state ? span_notice("You have pried the window into the frame.") : span_notice("You have pried the window out of the frame.")))
 
+	if(user.a_intent == INTENT_HARM)
+		return
```

Comment:
???

Reply:
Harm intent is handled on the parent call.